backend/api/handlers/survey/GetAfterSemesterSurveyHandler.go

```diff
@@ -4,12 +4,68 @@ import (
 	"fmt"
 	"github.com/gin-gonic/gin"
 	"net/http"
+	"errors"
+	"LiteracyLink.com/backend/api/model"
+	"LiteracyLink.com/backend/auth"
+	"LiteracyLink.com/backend/db"
+	"gorm.io/gorm"
 )
 
 func GetAfterSemesterSurveyHandler(c *gin.Context) {
-	userId := c.Param("user_id")
+	EOS_ID := c.Param("EOS_p_s_id")
 	c.JSON(http.StatusOK, gin.H{
 		"status":  "success",
-		"message": fmt.Sprintf("TODO: make func to check semester survey form for user: %s has been taken", userId),
+		"message": fmt.Sprintf("TODO: make func to get survey with ID: %s", EOS_ID),
+	})
+
```

Comment:
we don't need lines 16-19 here. should handle requests before sending a success message. can probably just delete it